spec/features/admin_series_create_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+describe 'the admin user flow', type: :feature do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.